test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -402,8 +400,11 @@ public Exception getException()
          */
         public void assertCleanStdErr()
         {
-            assertTrue("Failed because cleaned stdErr wasn't empty: " + getCleanedStderr(),
-                       getCleanedStderr().isEmpty());
+            String raw = getStderr();
+            String cleaned = getCleanedStderr();
+            assertTrue("Failed to clean stderr completely.\nRaw (length=" + raw.length() + "):\n" + raw + 
+                       "\nCleaned (length=" + cleaned.length() + "):\n" + cleaned,
+                       cleaned.trim().isEmpty());
```

Comment:
Not strictly necessary for the fix, but it makes clean stderr issues easier to debug.